diff --git a/mythtv/libs/libmythui/mythuihelper.h b/mythtv/libs/libmythui/mythuihelper.h
index d443057..37428d8 100644
--- a/mythtv/libs/libmythui/mythuihelper.h
+++ b/mythtv/libs/libmythui/mythuihelper.h
@@ -9,8 +9,8 @@
 #include "mythuiexp.h"
 #include "themeinfo.h"
 
-#define DEFAULT_UI_THEME "MythCenter"
-#define FALLBACK_UI_THEME "Terra"
+#define DEFAULT_UI_THEME "Mythbuntu"
+#define FALLBACK_UI_THEME "MythCenter-wide"
 
 class MythUIHelperPrivate;
 class MThreadPool;
