Index: mythtv/mythtv/libs/libmythui/mythuithemehelper.h
===================================================================
--- mythtv.orig/mythtv/libs/libmythui/mythuithemehelper.h	2015-04-12 15:24:03.598909276 +0000
+++ mythtv/mythtv/libs/libmythui/mythuithemehelper.h	2015-04-12 15:24:03.598909276 +0000
@@ -4,8 +4,8 @@
 // MythTV
 #include "themeinfo.h"
 
-static constexpr const char* DEFAULT_UI_THEME  { "MythCenter" };
-static constexpr const char* FALLBACK_UI_THEME { "Terra"      };
+static constexpr const char* DEFAULT_UI_THEME  { "MythCenter-wide" };
+static constexpr const char* FALLBACK_UI_THEME { "MythCenter" };
 
 class MUI_PUBLIC MythUIThemeHelper
 {
